client/me/get-apps/index.jsx

```diff
@@ -1,82 +1,104 @@
 /**
  * External dependencies
  */
-var React = require( 'react' );
-
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-var MeSidebarNavigation = require( 'me/sidebar-navigation' ),
-	Main = require( 'components/main' ),
-	Button = require( 'components/button' ),
-	Card = require( 'components/card' );
-
-module.exports = React.createClass( {
-
-	displayName: 'GetApps',
-
-	render: function() {
-		return (
-			<Main className="get-apps">
-				<MeSidebarNavigation />
-
-				<h1 className="get-apps__title">{ this.translate( 'Get Apps' ) }</h1>
-				<p className="get-apps__subheading">{ this.translate( 'Manage all your WordPress.com and Jetpack-enabled sites in one place. On your desktop, or on the go.' ) }</p>
+import MeSidebarNavigation from 'me/sidebar-navigation';
+import Main from 'components/main';
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
 
-				<h2 className="form-section-heading">{ this.translate( 'Desktop apps' ) }</h2>
-				<Card className="get-apps__desktop">
+const defaultProps = {
+  displayName: 'GetApps',
+};
+const propTypes = {
+  translate: PropTypes.func.isRequired,
+};
+const GetApps = ( { translate } ) => (
```

Comment:
I encountered an issue w/ GetApps as a named export. The code runs w/o the named export. I used the  `list-plan-upgrade-nudge.jsx` as an example to copy.